app/views/users/email_language/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<h1 class="h2"><%= t('account.email_language.edit_title') %></h1>
+
+<p class="margin-bottom-4">
+  <%= t('account.email_language.languages_list',
+        app_name: APP_NAME,
+        list: I18n.available_locales.
+          map { |locale| t("account.email_language.name.#{locale}") }.
+          to_sentence(last_word_connector: " #{t('account.email_language.sentence_connector')} ")) %>
+</p>
+
+<%= validated_form_for(current_user, url: account_email_language_path, method: 'PATCH') do |f| %>
+  <fieldset class="usa-fieldset">
+    <% label_id = SecureRandom.hex %>
+    <%= content_tag :label, t('account.email_language.please_select'), id: label_id %>
+    <%= render partial: 'shared/email_languages',
+               locals: {
+                 selection: current_user.email_language,
+                 labelledby: label_id,
+                 f: f
+               } %>
+  </fieldset>
+  <%= f.button :submit, t('forms.buttons.submit.default'), class: 'btn-wide col-6' %>
```

Comment:
FYI, at small viewport widths, the width-constraint of `col-6` can cause wrapping, depending on either the viewport width or locale.

Example:

![image](https://user-images.githubusercontent.com/1779930/97890978-34e9b580-1cfc-11eb-87e4-a0bd8e6a483c.png)

Reply:
Do you have a recommended width I should use instead?

Reply:
> Do you have a recommended width I should use instead?

Might to defer to @bpdesigns if he has any recommendations, but what we could do is use [USWDS responsive variants](https://designsystem.digital.gov/utilities/layout-grid/#responsive-variants) to allow more space at smaller viewports, then collapse back down at larger viewports.

I commented on the wrong line, but here:

https://github.com/18F/identity-idp/blob/215c5127125be859c7416e3b32f87210b5dd88bb/app/views/shared/_email_languages.html.erb#L12

That could be:

```html
      <li class="grid-col-8 mobile-lg:grid-col-6">
```

![responsive-variant mov](https://user-images.githubusercontent.com/1779930/97897355-5cdd1700-1d04-11eb-8cd6-5d97f810ec85.gif)

Alternatively, `flex-basis` + `flex-grow` could be a nice way to allow this to expand, though it'd take more work to think through how that would work with existing utilities and to make sure that every option resizes accordingly.

Reply:
Ok, I tried `grid-col-8 mobile-lg:grid-col-6` in 45d490684789897ce6abb1ae618639b275d9554f, and I updated the `<li>` as well as the submit buttons (it seemed weird to me that the submit buttons were so close to the same width as the items...but not matching, so I opted to make them the same)